src/xrpl-api.js

```diff
@@ -55,7 +61,7 @@ class XrplApi {
 
         this.#client.on('disconnected', (code) => {
             if (this.#autoReconnect && !this.#isPermanentlyDisconnected) {
-                console.log(`Connection failure for ${this.#rippledServer} (code:${code})`);
+                console.log(`Connection failure for ${this.#primaryRippledServer} (code:${code})`);
```

Comment:
Here we don't know which server is disconnected right? whether it's the primary or the fallback